src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is the use of this empty file?

Reply:
actually i have defined the whole thing in environment variable

Reply:
removed the application.properties file @manishatGit